libi2pd/NetDb.cpp

```diff
@@ -123,13 +126,19 @@ namespace data
 					m_Requests.ManageRequests ();
 					lastManageRequest = ts;
 				}
-				if (ts - lastSave >= 60) // save routers, manage leasesets and validate subscriptions every minute
+				if (ts - lastLeasesetsManage >= 60) // manage leasesets and validate subscriptions every minute
 				{
-					if (lastSave)
+					if (lastLeasesetsManage)
 					{
-						SaveUpdated ();
+						RemoveExpired ();
```

Comment:
Removing every minute?

Reply:
As the old `SaveUpdated` did.

Reply:
It must be with interval as specified in this config parameter. Otherwise you might end up with empty netdb quickly

Reply:
Sorry for the long delay. As far as I can see, doing it less frequently allows netdb to bloat uncontrollably. One apparent issue that needs to be fixed is that recently updated (`IsUpdated() == true`) but now unreachable records are deleted from netdb, which deviates from existing behaviour. Otherwise netdb service does the same thing, except for less frequent disk writes. If there is something else I fail to see, please, tell me.

Reply:
Also persisting netDb on exit does not work with the default systemd unit's stop signal - SIGQUIT.